src/praisonai/praisonai/bots/_delivery.py

```
@@ -0,0 +1,564 @@
+            await adapter.send_message(channel_id, content, **send_kwargs)
+            
+            # Success!
+            if attempt > 1:
+                logger.info(
+                    f"[{platform}] Delivery succeeded after {attempt} attempts "
+                    f"to channel {channel_id}"
+                )
+            
+            return True, None
+            
+        except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)